File: problem/mult1/9-12/main.py

# main.py: main file for "mult1" solutions
#
# Copyright (C) 2020  Rudy Matela
# All rights reserved.
#
#
# This file is part of the CSCX Online Judge.
#
# This is to be appended to the submitted file.

# standard input is already processed
# as the submitted programmed is run when imported

with open("in.txt") as filein:
  for line in filein:
    x,y,z = [int(x) for x in line.split()]
    print(mult(x,y,z))

File: problem/mult1/9-12/main.rb
# main.rb: main file for solutions to the mult problem
#
# This is appended to the submitted Ruby program and tests the mult function.
#
# The submitted file processes the standard input and this processes the
# "in.txt" file.
#
#
# Copyright (C) 2020-2021  Rudy Matela
# All rights reserved.

# standard input is already processed
# as the submitted programmed is run when imported
File.foreach("in.txt") do |line|
  x, y, z = line.split.map { |s| s.to_i }
  puts mult(x,y,z)
end
